packages/roosterjs-editor-dom/lib/selection/TableSelectionRange.ts

```diff
@@ -0,0 +1,24 @@
+import VTable from '../table/VTable';
+import { ITableSelectionRange, SelectionRangeTypes } from 'roosterjs-editor-types';
+/**
+ * Table selection Range used in the getSelectedRangeEx editor Api
+ * Can create a object with an array of ranges depending on a vTable range provided.
+ */
+export default class TableSelectionRange implements ITableSelectionRange {
```

Comment:
This class doesn't bring any value. No need to have it. TableSelectionRange is just a pure interface without any member function

Reply:
If need you can create a helper function to create instance of TableSelectionRange and calculate areAllCollapsed. The put it to the place where it is used, and no need to export it